test/torchaudio_unittest/transducer_test.py

```diff
@@ -241,6 +242,14 @@ def compute_with_pytorch_transducer(data):
     return costs, gradients
 
 
+def skipIfNoRNNT(test_item):
+    try:
+        _ = torch.ops.torchaudio.rnnt_loss
+        return test_item
+    except RuntimeError:
+        return unittest.skip("torchaudio C++ extension is not compiled with RNNT")(test_item)
```

Comment:
we could say "with the RNN transducer loss", or more explicitly:
```suggestion
        return unittest.skip("torchaudio C++ extension is not compiled with BUILD_TRANSDUCER=1")(test_item)
```

Reply:
@jieruan02 Please do not include `BUILD_TRANSDUCER=1` here.

* This could be changed anytime. When we update the build process, we update the documentations that we expect to find build process, which is README, CONTRIBUTING etc. A message in a test decorator is the place one would never expect to find a build direction so there is a high chance that this becomes obsolete and that case, it gives a bad developer experience.
*  It is not quite correct to say "skipping because it is not built with BUILD_TRANSDUCER=1". This message assumes that BUILD_TRANSDUCER=1 will build the required component, which is not wrong, but that is not guaranteed to be the only way. Why would give such an indirect, implied message, where the core of the logic is either it has certain feature compiled/available.

Reply:
Just say `torchaudio C++ extension is not compiled with the RNN transducer loss`